src/legacy/core_plugins/kibana/public/discover/doc_table/components/pager/tool_bar_pager_buttons.test.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { mountWithIntl, shallowWithIntl } from 'test_utils/enzyme_helpers';
+import { ToolBarPagerButtons } from './tool_bar_pager_buttons';
+
+test('it renders ToolBarPagerButtons', () => {
+  const props = {
+    hasPreviousPage: true,
+    hasNextPage: true,
+    onPageNext: jest.fn(),
+    onPagePrevious: jest.fn(),
+  };
+  const wrapper = shallowWithIntl(<ToolBarPagerButtons {...props} />);
+  expect(wrapper).toMatchSnapshot();
+});
+
+test('it renders ToolBarPagerButtons with clickable next and previous button', () => {
+  const props = {
+    hasPreviousPage: true,
+    hasNextPage: true,
+    onPageNext: jest.fn(),
+    onPagePrevious: jest.fn(),
+  };
+  const wrapper = mountWithIntl(<ToolBarPagerButtons {...props} />);
+  wrapper.find('[data-test-subj="btnPrevPage"]').simulate('click');
```

Comment:
FYI, EUI publishes a helper that can make this a little terser:

```js
import { findTestSubject } from '@elastic/eui/lib/test';
findTestSubject(wrapper, 'btnPrevPage').simulate('click');
```

Reply:
Very good to know :)
<img width="242" alt="Screenshot 2019-07-15 at 17 11 12" src="https://user-images.githubusercontent.com/1508364/61226904-98c12700-a723-11e9-9bc3-bebbc60d5fa8.png">

Reply:
very good hint, I've adapted the test